ComputerVision/cannyVariables.py

```diff
@@ -20,6 +22,7 @@ def change_low_threshold (lt) :
         settings.sliders["highthresholdslider"].set(settings.thresholdVariables["currentht"])
     gausimg = cv2.GaussianBlur(settings.images["originalphotowithCV"], (5,5), settings.thresholdVariables["currentsigma"])
     edges = cv2.Canny(gausimg, settings.thresholdVariables["currentlt"], settings.thresholdVariables["currentht"])
+    settings.images["latestCanny"] = edges
```

Comment:
![image](https://user-images.githubusercontent.com/22376976/110864933-cf509b80-8290-11eb-954c-aecc4761683b.png)
I can see that you had to add your change everywhere cause your code is duplicated in three pllaces

Reply:
I moved the code into a separate function to prevent this, thanks, also... what change?

Reply:
the settings.images["...